modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/FrontendDataSet.js

```diff
@@ -974,6 +980,7 @@ FrontendDataSet.defaultProps = {
 	inlineEditingSettings: null,
 	items: null,
 	itemsActions: null,
+	onSelect: () => {},
```

Comment:
perhaps meant `onSearch` here? or both?

Reply:
Nah, I did mean `onSelect` here. It's a side-effect of converting `MainSearch` to TS.

I would really love to see us taking one dedicated tech debt task, to convert `FrontendDataSet.js` to TS.